spec.bs

```diff
@@ -0,0 +1,19 @@
+<pre class="metadata">
+Title: FLEDGE
+Shortname: fledge
+Repository: WICG/turtledove
+Inline Github Issues: true
+Group: WICG
+Status: CG-DRAFT
+Level: 1
+URL: https://wicg.github.io/fledge/
+Boilerplate: omit conformance, omit feedback-header
+Editor: Paul Jensen, Google https://www.google.com/, pauljensen@google.com

```

Comment:
Now the build is failing because we're missing an Abstract line, how about:
```suggestion
Editor: Paul Jensen, Google https://www.google.com/, pauljensen@google.com
Abstract: Provides a privacy advancing API to facilitate interest group based advertising.
```